src/main/python/keycodes/keycodes_v6.py

```diff
@@ -592,6 +592,7 @@ class keycodes_v6:
 for x in range(16):
     keycodes_v6.kc["LT{}(kc)".format(x)] = keycodes_v6.kc["QK_LAYER_TAP"] | (((x) & 0xF) << 8)
 
+#TODO(userkeycodes): temp workaround, keycodes handling is messy, rework this later
```

Comment:
shouldn't be in v6 since the value is actually correct here

Reply:
> shouldn't be in v6 since the value is actually correct here

ohh, can you accept only the v5 file change? or should i redo it?

Reply:
Oh ok i figured out how to do remove it 👌